Gemfile.lock

```diff
@@ -1,23 +1,23 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.5)
+    addressable (2.8.6)
```

Comment:
Do we want these changes in the Gemfile.lock?

Reply:
Please ignore the changes in the gemfile.lock.